apps/nextjs-website/src/_contents/translations.ts

```diff
@@ -383,7 +383,7 @@ export const translations = {
         `Abbiamo inviato una e-mail a ${email}. Clicca sul bottone contenuto al suo interno per verificarla.`,
       didntReceiveEmail:
         "Non hai ricevuto l'e-mail? Controlla se nella posta indesiderata oppure",
-      resendEmail: 'Reinvia e-mail',
+      resendEmail: 'Reinvia email',
```

Comment:
Is there a reason why this copy should be `email` and other (like the one line above) should be `e-mail`?

Reply:
All other occurencies of `e-mail` should have been replaced in the next PR #605